src/WidgetBase.ts

```diff
@@ -626,9 +701,45 @@ export class WidgetBase<P extends WidgetProperties = WidgetProperties, C extends
 			return this.dNodeToVNode(child);
 		});
 
+		this._prepareDeferredProperties(dNode);
+
 		return dNode.render();
 	}
 
+	private _prepareDeferredProperties(node: DNode) {
```

Comment:
Again feels like this is a change that is actually unrelated to the meta config changes.